tests/test_vav_turndown_during_reheat.py

```diff
@@ -0,0 +1,128 @@
+            datetime(2024, 8, 1, 13, 0, 0),
+            datetime(2024, 8, 1, 14, 0, 0),
+            datetime(2024, 8, 1, 15, 0, 0),
+        ]
+
+        data = [
+            [True, 350, 620],
+            [True, 370, 620],
+            [False, 360, 620],
+            [False, 380, 620],
+        ]
+
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        verification_obj = run_test_verification_with_data(
+            "VAVTurndownDuringReheat", df
+        )
+
+        results = pd.Series(list(verification_obj.result))
+        expected_results = pd.Series(
+            [
+                True,
+                True,
+                True,
+                True,
+            ]
+        )
+        self.assertTrue(results.equals(expected_results))
+
+        binary_result = verification_obj.check_bool()
+        self.assertTrue(binary_result)
+
+    def test_vav_turndown_during_reheat_fail(self):
+        points = [
+            "reheat_coil_flag",
+            "V_dot_VAV",
+            "V_dot_VAV_max",
+        ]
+
+        timestamp = [
+            datetime(2024, 8, 1, 12, 0, 0),
+            datetime(2024, 8, 1, 13, 0, 0),
+            datetime(2024, 8, 1, 14, 0, 0),
+            datetime(2024, 8, 1, 15, 0, 0),
+        ]
+
+        data = [
+            [False, 350, 620],
+            [False, 370, 620],
+            [True, 360, 620],
+            [True, 380, 620],
+        ]
+
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        verification_obj = run_test_verification_with_data(
+            "VAVTurndownDuringReheat", df
+        )
+
+        results = pd.Series(list(verification_obj.result))
+        expected_results = pd.Series(
+            [
+                False,
+                False,
+                False,
+                False,
+            ]
+        )
+        self.assertTrue(results.equals(expected_results))
+
+        binary_result = verification_obj.check_bool()
+        self.assertFalse(binary_result)
+
+    def test_vav_turndown_during_reheat_untested(self):
+        points = [
+            "reheat_coil_flag",
+            "V_dot_VAV",
+            "V_dot_VAV_max",
+        ]
+
+        timestamp = [
+            datetime(2024, 8, 1, 12, 0, 0),
+            datetime(2024, 8, 1, 13, 0, 0),
+            datetime(2024, 8, 1, 14, 0, 0),
+            datetime(2024, 8, 1, 15, 0, 0),
+        ]
+
+        data = [
+            [False, 350, 620],
+            [False, 370, 620],
+            [False, 360, 620],
+            [False, 380, 620],
+        ]
+
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        verification_obj = run_test_verification_with_data(
+            "VAVTurndownDuringReheat", df
+        )
+        results = list(verification_obj.result)
+        expected_results = [
+            "Untested",
+            "Untested",
+            "Untested",
+            "Untested",
+        ]
+
+        self.assertEqual(results, expected_results)
```

Comment:
check bool results, also, maybe all add additional tests on integer value - calculated ratios. For instance, 1/3

Reply:
I believe this isn't needed based on our discussion (1/3 = 0.333 not 0.0 in Python 3). But, based on your comment in PR # 55, I added the following code.

```Python
self.assertEqual(verification_obj.check_bool(), "Untested")
```